docs/ipv4_forwarding.md

```diff
@@ -0,0 +1,32 @@
+## IPv4 Forwarding
+
+To enable PSO pods to communicate with each other across nodes in the cluster IPv4 forwarding must be enabled on all cluster nodes.
+
+If forwarding is not enabled PSO will fail to install and you will see a number of PSO related pods going into `CrashLoopBackoff` mode.
+
+### Check current IPv4 forwarding
+
+To check the current state:
+
+```bash
+# sysctl net.ipv4.ip_forward
+net.ipv4.ip_forward = 1
+```
+
+The output number `1` indocates that IPv4 forwarding is enabled. 
+If the output number is `0` IPv4 forwarding is disabled and must be enabled.
+
+### Enable IPv4 Forwarding
+
+If IPv4 is not enabled you must enable it as follows on each cluster node:
+
+```bash
+# sysctl -w net.ipv4.ip_forward=1
```

Comment:
Uncomment it, as this is the command to enable it?

Reply:
It's not a comment - it's to identify this is a command line and the following line is the output from the command